packaging/pkg_helpers.bash

```diff
@@ -173,6 +173,7 @@ setup_pip_pytorch_version() {
   else
     pip_install "torch==$PYTORCH_VERSION$PYTORCH_VERSION_SUFFIX" \
       -f https://download.pytorch.org/whl/torch_stable.html \
+      -f https://download.pytorch.org/whl/test/torch_test.html \
```

Comment:
Same goes for these lines, might be better if you just `upload_channel` here as well